integration_test/cloudbuild-integration-test.yaml

```diff
@@ -420,6 +515,7 @@ substitutions:
   _LOCATION: us-central1
   _DATASET: healthcare-dicom-dicomweb-adapter-test
   _STORE_NAME: integration-test-store
+  _BUCKET: healthcare-dicom-dicomweb-adapter-integration-test
```

Comment:
Please change this to a different bucket name like "dicomweb-adapter-integration-test" so that I can create the bucket in our integration test project.